FirebaseAdmin/FirebaseAdmin/Auth/UserRecordArgs.cs

```diff
@@ -265,6 +242,29 @@ private static string CheckCustomClaims(IReadOnlyDictionary<string, object> clai
             return customClaimsString;
         }
 
+        internal CreateUserRequest ToCreateUserRequest()
+        {
+            return new CreateUserRequest(this);
+        }
+
+        internal UpdateUserRequest ToUpdateUserRequest()
+        {
+            return new UpdateUserRequest(this);
+        }
+
+        private static string CheckPassword(string password)
+        {
+            if (password != null)
+            {
+                if (password.Length < 6)
+                {
+                    throw new ArgumentException("Password must be at least 6 characters long.");
+                }
+            }
```

Comment:
I wonder if instead the conditional can be:

```
if (password != null && password.Length < 6) 
{
    throw new ArgumentException("Password must be at least 6 characters long.");
}
```

Reply:
I think you can. `&&` will short-circuit and will not evaluate the second operand if password is `null`.

Reply:
Are `null` passwords allowed here?

Reply:
I think that null passwords can be passed into the `UserRecordArgs` constructor, maybe for a user without a password

I didn't write this chunk of code, I just happened to come across it when I was shuffling it to address linter comments and thought I'd add the `&&` to the ifs :) it's not very related to my actual changes otherwise